packages/web-app-files/src/components/SideBar/FileInfo.vue

```diff
@@ -51,6 +54,17 @@ export default {
   computed: {
     ...mapGetters(['capabilities']),
     modificationTime() {
+      if (this.isTrashbinRoute) {
+        return DateTime.fromRFC2822(this.file.mdate)
```

Comment:
```suggestion
        return DateTime.fromRFC2822(this.file.ddate)
```
Also, please use the helper mentioned in the other comment, if you decide to build it.

Reply:
Yep, good idea, thanks!